flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/checkpoints/CheckpointConfigInfo.java

```diff
@@ -57,6 +57,8 @@
 
 	public static final String FIELD_NAME_UNALIGNED_CHECKPOINTS = "unaligned_checkpoints";
 
+	public static final String FIELD_NAME_TOLERABLE_FAILED_CHECKPOINT = "tolerable_failed";
```

Comment:
how about changes below?
```suggestion
	public static final String FIELD_NAME_TOLERABLE_FAILED_CHECKPOINTS = "tolerable_failed_checkpoints";
```